### PR TITLE
Adds optional extended outputFormat

### PR DESCRIPTION
**Pull request by dcolligan**
Issue #107

### CODE REVIEW

File: ga4gh/cli.py [hunk 1]
```diff
@@ -364,7 +388,10 @@ def client_main(parser=None):
         "--workarounds", "-w", default='', help="The workarounds to use")
     parser.add_argument(
         "--key", "-k", help="The auth key to use")
-    subparsers = parser.add_subparsers(title='subcommands',)
+    parser.add_argument(
+        "--outputFormat", "-o", help="The output format to use",
+        default="minimal", choices=["minimal", "extended"])
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: A boolean flag would seem more natural here. What about -O, --minimalOutput, default=False? We keep the semantics we  have from before then and we can just add the --minimalOutput flag for your testing script.

**Comment by dcolligan** [comment]
@jeromekelleher pushed a commit addressing your concerns

File: ga4gh/cli.py [hunk 2]
```diff
@@ -153,7 +154,30 @@ def __init__(self, args):
         self.setHttpClient(request, args)
 
     def run(self):
-        self._run('searchVariants', 'id')
+        if self._minimalOutput is True:
```
Comment (jeromekelleher) [inline ga4gh/cli.py]: `if self._minimalOutput: ` is the canonical way of testing a boolean.

File: scripts/google_examples.py [hunk 3]
```diff
@@ -33,19 +34,24 @@ def runTests():
...
         }
         cmd = cmdStr.format(**cmdDict)
+        splits = cmd.split()
+        cleanCmd = ' '.join(splits)
         print(separator)
-        print(cmd)
+        print(cleanCmd)
         print(separator)
-        splits = cmd.split()
         proc = subprocess.Popen(splits)
```
Comment (jeromekelleher) [inline scripts/google_examples.py]: `subprocess.check_call(splits)` would be a slightly cleaner way of doing this.

**Comment by jeromekelleher** [comment]
+1 modulo nits. I think we should follow this up with something a bit more generic which gives useful output for all the other commands when #108 is done. Thanks @dcolligan.

File: scripts/google_examples.py (hunk 3 above)
Comment (dcolligan) [inline scripts/google_examples.py]: Yes, but that way suppresses all the output of the command, which we don't want.

Comment (dcolligan) [inline scripts/google_examples.py]: Nevermind, my mistake.

**Comment by dcolligan** [comment]
@jeromekelleher pushed commit that addressed nits

**Pull request merged**